diff --git a/.env b/.env
index f012497..e705778 100644
--- a/.env
+++ b/.env
@@ -19,7 +19,7 @@ DOCKER_NETWORK_NAME=jupyterhub-network
 DOCKER_NOTEBOOK_IMAGE=jupyter/minimal-notebook:e1677043235c

 # the local image we use, after pinning jupyterhub version
-LOCAL_NOTEBOOK_IMAGE=jupyterhub-user
+LOCAL_NOTEBOOK_IMAGE=jupyterhub-image-ifcopenshell

 # Notebook directory in the container.
 # This will be /home/jovyan/work if the default
